<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>581</width>
      <height>489</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Precio de Alfajor de Dulce de Leche:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Precio de Alfajor Frutal:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Precio de Alfajor de Mousse:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QDoubleSpinBox" name="SpinDLL">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="SpinF">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="SpinM">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Hynings Handwriting</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Ingrese su nombre:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineNombre"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>A little sunshine</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MENU DE CAJAS</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Caja de 6 (Sin descuento)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Caja de 12 (descuento del %10)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Caja de 24 (descuento del %20)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="bt6">
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt12">
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt24">
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>A little sunshine</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SABOR DE LA CAJA</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Dulce de leche</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Frutal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <family>Hynings Handwriting</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mousse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="btDulce">
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btFrutal">
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btMousse">
             <property name="text">
              <string>Seleccionar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btConfirmar">
       <property name="text">
        <string>Confirmar compra</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
